packages/widget-message-meet/src/containers/connected-activity-list/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,4 @@
+exports[`ConnectedActivityList renders properly 1`] = `
+<div
+  className="activity-list" />
+`;
```

Comment:
The mock store doesn't have any. I updated the actual activity list to have proper data in its tests. I wasn't sure about building out a fully populated store, but we can.